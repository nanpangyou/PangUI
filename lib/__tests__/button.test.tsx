// import React from "react";
// import Button from "../button";
// import renderer from "react-test-renderer";

// describe("Button", () => {
//   it("is a div Element", () => {
//     const json = renderer.create(<Button />).toJSON();
//     expect(json).toMatchSnapshot();
//   });
// });
describe("xx", () => {
  it("test", () => {
    expect(1).toEqual(1);
  });
});
