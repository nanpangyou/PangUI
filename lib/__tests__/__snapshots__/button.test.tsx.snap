// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button is a div Element 1`] = `
<div>
  按钮
</div>
`;
